docs/templates/nextflow_dsl2/nextflow.config

```diff
@@ -12,8 +12,7 @@ params {
   input                      = ''
 
   // Options: skipping process(es)
-//  skip_miso                  = false
-//  skip_qapa                  = false
+  params.run_benchmarking_event1 = flase
```

Comment:
typo: "flase"

Reply:
Thanks for catching this! I have fixed the typo